oat-sa/extension-tao-itemqti-pci
# Feature/tr 508 convert mathentryinteraction to ims compliant

Related to https://oat-sa.atlassian.net/browse/TR-508

-  Move `mathEntryInteraction` to ims folder
- Create factory from `mathEntryInteraction` and register ims api
- Make fonts inline into css. (I kept only woff, because others are optional and they are ~600k / file)
- Remove mathjax and tooltip dependency. They will be solved in another ticket
- Create migration script and update install script
- Adapt tests to interaction api

Requires:
- [x] https://github.com/oat-sa/extension-tao-itemqti/pull/1627

Test:
- run tests in `tao/views/build` with `npx grunt connect:test qtiitempcitest`. It should not fail. Please be sure you integrated https://github.com/oat-sa/extension-tao-itemqti/pull/1627 tao qti item runner into your tao, because it needs for the correct behavior.
- Install develop branch. Checkout this branch and run taoUpdate. PCI 1.0.0 version should be registered and should work correctly.
- Install this branch directly. PCI 1.0.0 version should be registered and should work correctly.
- Check creator part and modify interaction parameters, then run preview mode.
- Create a test with 2 mathEntryInteraction and check the factory works correctly and they do not listen to other's events.
- Check if you move to another item and move back to this interaction, the response is saved correctly and restored correctly. There is a rendering issue without https://github.com/oat-sa/tao-test-runner-qtinui-fe/pull/216, so be sure you will test with this.
- Check the same in new test runner.
- In new test runner, open overview that destroys interaction and check the state restore when you close it. https://github.com/oat-sa/tao-item-runner-qtinui-fe/pull/309 version of item runner is necessary for the correct behavior.
